File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/dart/practicas_3/biografia.dart
import 'dart:io';

void main () {

  print('Cuantas personas quieren dar a conocer su vida:');
  int cuantos = int.parse(stdin.readLineSync()!);

  for (int vueltas = 1; vueltas <= cuantos; vueltas++) {

    print('----------------------------------------------------------');

    print('!!ANTES QUE TODO DIGITAR LA INFORMACION REQUERIDA!!');

    print('----------------------------------------------------------');

    print('Nombres y apellidos:');
    String name = stdin.readLineSync()!;

    print('Edad:');
    String age = stdin.readLineSync()!;

    print('Ahora si porfavor diganos como fue su vida:');
    String life = stdin.readLineSync()!;

    print('----------------------------------------------------------');    

    print('INFORMACION PROCESADA Y ORDENADA DEL N: $vueltas');

    print('----------------------------------------------------------');

    print('La persona identifica como $name de $age años, nos va a relatar hoy su vida acontinuacion.');
    print(' ');
    print('BIOGRAFIA:');
    print(' ');
    print(life);

    print('----------------------------------------------------------');
  }

}

File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/ruby/practicas_3/biografia.rb

puts "Cuantas personas quieren dar a conocer su vida:"
cuantos = gets.chomp.to_i

  for vueltas in 1..cuantos

    puts "----------------------------------------------------------"

    puts "!!ANTES QUE TODO DIGITAR LA INFORMACION REQUERIDA!!"

    puts "----------------------------------------------------------"

    puts "Nombres y apellidos:"
    name = gets.chomp

    puts "Edad:"
    age = gets.chomp.to_i

    puts "Ahora si porfavor diganos como fue su vida:"
    life = gets.chomp

    puts "----------------------------------------------------------"

    puts "INFORMACION PROCESADA Y ORDENADA DEL N: #{vueltas}"

    puts "----------------------------------------------------------"

    puts "La persona identifica como #{name} de #{age} años, nos va a relatar hoy su vida acontinuacion."
    puts " "
    puts "BIOGRAFIA:"
    puts " "
    puts life

    puts "----------------------------------------------------------"

  end